Bottom: 016cbbf35c4236dccf56097d91311ce00713e1a4
Top:    e981ba4220f641cd1031340991710b8244789d29
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 02:54:34 +0000

Refresh of draw-c-support-for-coloring-ob

---

diff --git a/src/undo.h b/src/undo.h
index c52b051..18117cb 100644
--- a/src/undo.h
+++ b/src/undo.h
@@ -32,8 +32,9 @@
 
 #include "global.h"
 
-#define DRAW_FLAGS	(RATFLAG | SELECTEDFLAG | SQUAREFLAG \
-			| HIDENAMEFLAG | HOLEFLAG | OCTAGONFLAG | CONNECTEDFLAG | FOUNDFLAG | CLEARLINEFLAG)
+#define DRAW_FLAGS  (RATFLAG | SELECTEDFLAG | SQUAREFLAG |    \
+                     HIDENAMEFLAG | HOLEFLAG | OCTAGONFLAG |  \
+                     CONNECTEDFLAG | FOUNDFLAG | CLEARLINEFLAG)
 
 											/* different layers */
